aspnetcore/razor-components/components.md

```diff
@@ -479,6 +479,32 @@ The [sample app](https://github.com/aspnet/Docs/tree/master/aspnetcore/razor-com
 
 The base class should derive from `ComponentBase`.
 
+## Importing components
+
+Components infer their namespace based on the file path and the file name.
+Example,
+`ComponentsSample/Pages/Index.razor` will be in the namespace `ComponentsSample.Pages`. So, all components in the same folder `Pages` and the parent folder `ComponentsSample` are already in scope.
```

Comment:
I wouldn't start with this example. I would say that the bindings between tag names and component classes are based on usings. When you explain these concepts in terms of normal C# name binding rules they make much more sense.